.circleci/scripts/.tests.env

```diff
@@ -14,7 +14,7 @@ export ANDROID_SDK_BUILD_API_LEVEL="28"
 # Google APIs for Android level
 export ANDROID_GOOGLE_API_LEVEL="23"
 # Minimum Android API Level we target
-export ANDROID_SDK_TARGET_API_LEVEL="19"
+export ANDROID_SDK_TARGET_API_LEVEL="23"
```

Comment:
Shouldn't this be inherented from [build.gradle](https://github.com/facebookincubator/spectrum/blob/master/build.gradle#L11)?

Reply:
I'm not sure to be honest. This is the emulator version we want to use for testing. It seems reasonable to match targetSdkVersion, but we're on v28 there so I'd prefer to not lower that.